x-pack/plugin/esql/qa/server/src/main/java/org/elasticsearch/xpack/esql/qa/rest/EsqlSpecTestCase.java

```diff
@@ -206,23 +227,17 @@ protected static void checkCapabilities(RestClient client, TestFeatureService te
                 throw e;
             }
         }
-
-        var features = Stream.concat(
-            new EsqlFeatures().getFeatures().stream(),
-            new EsqlFeatures().getHistoricalFeatures().keySet().stream()
-        ).map(NodeFeature::id).collect(Collectors.toSet());
-
-        for (String feature : testCase.requiredCapabilities) {
-            var esqlFeature = "esql." + feature;
-            assumeTrue("Requested capability " + feature + " is an ESQL cluster feature", features.contains(esqlFeature));
-            assumeTrue("Test " + testName + " requires " + feature, testFeatureService.clusterHasFeature(esqlFeature));
-        }
+        return false;
     }
 
     protected boolean supportsInferenceTestService() {
         return true;
     }
 
+    protected boolean supportsIndexModeLookup() throws IOException {
```

Comment:
I guess you didn't mean on having this method to `throws IOException`?

Reply:
Not originally, but the capabilities check in the sub-classes can throw the exception, so the parent signature needed it too.